.github/workflows/deploy.yml

```
@@ -34,32 +34,37 @@ jobs:
-      - uses: actions/checkout@v3
-      - name: Set up Python
-        uses: actions/setup-python@v4.3.0
+      - uses: actions/checkout@v4
+      - name: Set Dockerfile name
+        if: matrix.image.name != 'latest'
+        run: echo "DOCKERFILE_NAME=${{ matrix.image.name }}" >> $GITHUB_ENV
+      - name: Set Dockerfile name latest
+        if: matrix.image.name == 'latest'
+        run: echo "DOCKERFILE_NAME=python${{ matrix.image.python_version }}" >> $GITHUB_ENV
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
```

actionlint (1.7.11)

[error] the runner of "docker/setup-buildx-action@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)